packages/fxa-settings/src/components/ThirdPartyAuth/index.tsx

```diff
@@ -0,0 +1,80 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+export type ThirdPartyAuthProps = {
+  enabled?: boolean;
+};
+
+const ThirdPartyAuth = ({ enabled = false }: ThirdPartyAuthProps) => {
+  if (!enabled) {
+    return null;
+  }
+
+  return (
+    <>
+      <div className="flex flex-col">
+        <div className="text-sm flex items-center justify-center my-6">
+          <div className="flex-1 h-px bg-grey-300 divide-x"></div>
+          <div className="mx-4 uppercase">OR</div>
```

Comment:
I missed this, agreed with @pdehaan, this needs to be localized.

Reply:
Also noting, since without context this would be very difficult to localize, so please include a comment that indicates where/how this will appear.
As an example (please create your own or adjust below as appropriate):
```
# This appears when a user has the option to authenticate via third party accounts in addition to their Firefox account. 
# Firefox account login appears on top, and third party options appear on bottom. 
# This string appears as a separation between the two, in the following order: "Enter your password" "Or"(this string) "Continue with Google"(continue-with-google-button) / "Continue with Apple"(continue-with-apple-button)
```

Reply:
@bcolsson Re: `<div className="mx-4 uppercase">OR</div>`, I also noticed there is a suspicious looking [`uppercase` class](https://tailwindcss.com/docs/text-transform#transforming-text) in there.
Does that work for localized content, or are there some locales that would have issues w/ converting strings to uppercase?

Possibly not an issue though since I see about 13 other places in the codebase where we're using either the `uppercase` Tailwind class (8&times;) or `text-transform: uppercase;` (5&times;) directly:

<details>
<summary><kbd>git grep "uppercase" packages | grep -E "(text-transform:|className)"</kbd></summary>

```sh
packages/fxa-admin-panel/src/components/Nav/index.tsx:      <h2 className="mb-3 uppercase text-sm tracking-wide font-normal text-grey-500">
packages/fxa-admin-panel/src/components/PageAccountSearch/index.tsx:          className="block uppercase text-sm text-grey-500"
packages/fxa-auth-server/lib/senders/emails/storybook.css:  text-transform: uppercase;
packages/fxa-content-server/app/styles/modules/_custom-rows.scss:    text-transform: uppercase;
packages/fxa-content-server/app/styles/modules/_input-row.scss:      text-transform: uppercase;
packages/fxa-content-server/app/styles/modules/_input-row.scss:      text-transform: uppercase;
packages/fxa-content-server/app/styles/modules/_settings-totp.scss:      text-transform: uppercase;
packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx:        <h2 className="text-xs text-grey-400 font-semibold uppercase">
packages/fxa-settings/src/components/Settings/UnitRowSecondaryEmail/index.tsx:                    className="uppercase block text-orange-600 font-bold text-xs"
packages/fxa-settings/src/components/ThirdPartyAuth/index.tsx:          <div className="mx-4 uppercase">OR</div>
packages/fxa-settings/src/components/WarningMessage/index.tsx:          span: <span className="text-red-600 uppercase">{warningType}</span>,
packages/fxa-settings/src/components/WarningMessage/index.tsx:          <span className="text-red-600 uppercase">{warningType}</span>{' '}
packages/fxa-settings/src/pages/Signin/CompleteSignin/index.tsx:              <span className="text-red-600 uppercase">
```
</details>

Reply:
@pdehaan - This is a good call. It's possible we didn't notice this being used in the past. It's best to avoid running uppercase transform here, as different languages have different stylistic rules that could conflict. (Example, Greek doesn't use accents when block capitals, but uppercase would preserve accents.)

It'd be better to leave the style in the string itself and let translators apply the appropriate style themselves.